src/docs/getting-started/upgrading.md

```diff
@@ -281,6 +294,10 @@ If you need to replace swup's internal handler for a custom implementation, don'
 instance method. Instead, specify that your hook handler should replace the internal one.
 
 ```js
-this.swup.replaceContent = () => { /* custom implementation */ }; // [!code --]
-this.swup.hooks.replace('content:replace', () => { /* custom implementation */ }); // [!code ++]
+this.swup.replaceContent = () => {
```

Comment:
We need to check these, auto-linting broke the diff blocks

Reply:
oh

Reply:
Reverted the linting